lib/jekyll-admin/server.rb

```diff
@@ -59,7 +59,8 @@ def front_matter
 
     def document_body
       body = if front_matter && !front_matter.empty?
-               YAML.dump(front_matter).strip
+               YAML.dump(restored_front_matter).strip
+                 .gsub(": 'null'", ": null") # restore null values
```

Comment:
With the above patch, it becomes:
`{"layout": null, "foo": false}` ==>`{"layout" => "nil", "foo" => false}` ==> `"---\nlayout: 'null'\nfoo: false\n---\n"` ==>
```
---
layout: null
foo: false
---
```